modules/sharedaddy/sharing-sources.php

```diff
@@ -1713,7 +1713,7 @@ public function get_name() {
 	}
 
 	public function get_display( $post ) {
-		return $this->get_link( 'https://api.whatsapp.com/send?text=' . rawurlencode( $this->get_share_title( $post->ID ) ) . ' ' . rawurlencode( $this->get_share_url( $post->ID ) ), _x( 'WhatsApp', 'share to', 'jetpack' ), __( 'Click to share on WhatsApp', 'jetpack' ) );
+		return $this->get_link( esc_url('https://api.whatsapp.com/send?text=' . rawurlencode( $this->get_share_title( $post->ID ) ) . ' ' . rawurlencode( $this->get_share_url( $post->ID ) )), _x( 'WhatsApp', 'share to', 'jetpack' ), __( 'Click to share on WhatsApp', 'jetpack' ) );
```

Comment:
Can we instead put the space concatenation inside `rawurlescape` and avoid wrapping the whole thing into another escaping call?

Reply:
Definitely. I'll go ahead and make that change. Thanks for the input!